config/samples/ocs_v1_storageclusterpeer.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: ocs.openshift.io/v1
+kind: StorageClusterPeer
+metadata:
+  labels:
+  name: storageclusterpeer-sample
+spec:
+  ocsApiServerUri: 10.0.0.0/31659
```

Comment:
intentionally mentioned `/` inplace of `:`?

Reply:
No, updated it. thanks for pointing it out